front/src/modules/scenario/components/AddOrEditScenarioModal.tsx

```diff
@@ -201,13 +203,10 @@ export default function AddOrEditScenarioModal({
     }
   };
 
-  useEffect(() => {
-    setCurrentScenario({ ...currentScenario, infra_id: infraID });
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [infraID]);
```

Comment:
We need to keep this, otherwise the infra selected by the user won't be added to the scenario data and we can't submit the new scenario

Reply:
Done